src/c_api/c_api.cc

```diff
@@ -24,82 +24,6 @@
 
 
 namespace xgboost {
-// booster wrapper for backward compatible reason.
```

Comment:
Is this whole class redundant?

Reply:
I don't know what it does.  @CodingCat is there any "compatible reason" in distributed env?